crowbar_framework/lib/crowbar/conduit_resolver.rb

```diff
@@ -328,14 +328,14 @@ def cr_node_roles
 
     ## Return the DMI system attributes from the node
     def cr_dmi_system
-      return {} if @node[:dmi].nil? || @node[:dmi][:system].nil?
+      return {} if @node.automatic_attrs[:dmi].nil? || @node.automatic_attrs[:dmi][:system].nil?
```

Comment:
Using symbols here doesn't seem to work when loading a node from json. At least for me ```network-json-resolve --network-json network-config-default.json --node-json node.json bus_order <node>``` always return "No bus order defined for  ...." until I use strings instead of symbols here and in the following lines